EpicLoot/src/Magic/MagicItemEffects/ModifyAttackHealth.cs

```diff
@@ -0,0 +1,21 @@
+﻿using HarmonyLib;
+
+namespace EpicLoot.MagicItemEffects
+{
+    [HarmonyPatch(typeof(Attack), nameof(Attack.GetAttackHealth))]
+    public class ModifyAttackHealth_Attack_GetAttackHealth_Patch
+    {
+        public static void Postfix(Attack __instance, ref float __result)
```

Comment:
This could be a Prefix with a bool return type to skip running the original code if this new calculation is run in it's place.

Reply:
Fixed in: https://github.com/OrianaVenture/Randy_Vapok_ValheimMods/pull/10/commits/502502409e1382f19dc714aa91c7722d819f3fe3